ui/app/templates/partials/secret-form-show.hbs

```diff
@@ -1,4 +1,14 @@
-{{#if (and isV2 modelForData.destroyed)}}
+{{#if isWriteWithoutRead}}
+  <EmptyState
+    data-test-write-without-read-empty-message
+    @title="You do not have permission to read this secret."
+    @message={{if isV2
+      "A new version of this secret may be written but your associated policies do not allow you to read its current contents."
```

Comment:
Also a nit, but I don't think you need the word `associated` here 😬

Reply:
if you'd like, you could follow the sentence structure on line 7, like:

"Your policies permit you to write a new version of this secret, but do not allow you to read its current contents."

Reply:
Heh, not a nit at all - I also dig following a similar structure for the messages 👍